.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!---
+PLEASE READ:
+
+Unless you want to skip tests (//skip-ci):
+1) do not directly request review from anyone
+2) use 'Draft' PR until the tests pass and you are satisfied with all the content - then mark the PR as "Ready for review"
+-->
```

Comment:
Good idea, but since we are already creating this file, could you also add a reminder on how to run a particular tag? I always have to go back and check if it's really backticks and if they need to be around the whole thing or just the tag etc.

Reply:
@asmigala wdyt now?